src/scripts/quests/QuestLineHelper.ts

```diff
@@ -1125,6 +1125,76 @@ class QuestLineHelper {
         App.game.quests.questLines().push(ancientGolemsQuestLine);
     }
 
+ public static createDetectivePikachuQuestLine() {
```

Comment:
Could you move all the Detective Pikachu quest stuff in between Celebi and Hoenn quests? To stay consistent with the organization of this file. 
Same with the `loadQuestLines()` below.